packages/plugin-flex/src/sub-commands/flex-plugin.ts

```diff
@@ -673,10 +682,10 @@ export default class FlexPlugin extends baseCommands.TwilioClientCommand {
    * The command parse override
    */
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  protected parse<F, A extends { [name: string]: any }>(
+  protected async parseCommand<F, A extends { [name: string]: any }>(
```

Comment:
What are we renaming this? Let's keep it as `parse`

Reply:
Same fn definition is present is TwilioClientCommand class which is FlexPlugin's parent class. So typescript is throwing an error. Should we ignore it?